book/verifying-proofs/solidity-and-evm.md

```diff
@@ -9,9 +9,12 @@ By default, the proofs generated by SP1 are not verifiable onchain, as they are
 
 **The PLONK Bn254 prover is only guaranteed to work on official releases of SP1.**
 
-To use PLONK proving & verification locally, enable the `plonk` feature flag in the sp1-sdk and ensure that Go >1.22.1 is installed.
+To use PLONK proving & verification locally, ensure that you have Docker installed.
+
+If you would like to run the PLONK prover directly without Docker, you must have Go 1.22 installed and enable the `native-plonk` feature in `sp1-sdk`.
+
 ```toml
-sp1-sdk = { features = ["plonk"] }
+sp1-sdk = { features = ["native-plonk"] }
```

Comment:
You should move the second section under a <NOT RECOMMENDED> flag, doesn't seem like most users will need this.